src/engraving/dom/fermata.cpp

```diff
@@ -47,6 +47,7 @@ namespace mu::engraving {
 
 static const ElementStyle fermataStyle {
     { Sid::fermataPosAbove, Pid::OFFSET },
+    { Sid::fermataPosBelow, Pid::OFFSET },
```

Comment:
I'm not sure this is correct. According to the implementation of `initElementStyle`, this would mean that the `OFFSET` property is set to the value of `fermataPosBelow` (because that comes last) while the fermata actually has default placement above.

See also `getPropertyStyle`, which should make it unnecessary to list `fermataPosBelow` here.

Reply:
So it isn't really an initialization but a dummy placeholder?

Reply:
Leaving a comment about that?